recipe/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "dtumathtools" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f9fe7e34dc5be3954f451ea639c3ab47ba31c0b2b620f5e29618f1640bce1782
+
+build:
```

Comment:
Can this package be `noarch: python`? I don't see any compiled pieces. 

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
thanks a lot for your comments, sorry about the directory stuff.

Indeed the `noarch: python` may be applied here. I have added that, and removed the `requirements.build` for the `mac` stuff... Fingers cross that it works.